packages/@react-stately/utils/test/useControlledState.test.js

```diff
@@ -68,7 +68,8 @@ describe('useControlledState tests', function () {
 
     let TestComponent = (props) => {
       let [state, setState] = useControlledState(props.value, props.defaultValue, props.onChange);
-      useEffect(() => renderSpy());
+      // TODO should this really have `, []` so that the count that is asserted is 1 ?
+      useEffect(() => renderSpy(), []);
```

Comment:
state should probably be in this array
the idea behind this test, if i remember correctly, was to make sure that this syntax `setState((prev) => prev + 1)}` didn't cause a double render

that said, i'm really not remembering why it was only 1 call, it should call render spy on mount, and then once more when the button is clicked

Reply:
-> a3a905f